python/arcticdb/storage_fixtures/utils.py

```diff
@@ -0,0 +1,105 @@
+"""
+Copyright 2023 Man Group Operations Limited
+
+Use of this software is governed by the Business Source License 1.1 included in the file LICENSE.txt.
+
+As of the Change Date specified in that file, in accordance with the Business Source License, use of this software will be governed by the Apache License, version 2.0.
+"""
+
+import multiprocessing
+import subprocess
+import os
+import platform
+import requests
+import signal
+import socketserver
+import time
+import warnings
+from typing import Union, Any
+from contextlib import contextmanager
+
+_WINDOWS = platform.system() == "Windows"
+
+
+def get_ephemeral_port():  # https://stackoverflow.com/a/61685162/
```

Comment:
This seems much the same as the old way? https://github.com/man-group/ArcticDB/blob/615edcb76216024d51f2ec5ede92eec6ea52e587/python/tests/conftest.py#L86

Reply:
It's move from conftest